pkg/maps/worldcidrsmap/worldcidrs.go

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package worldcidrsmap
+
+import (
+	"net"
+	"unsafe"
+
+	"github.com/cilium/cilium/pkg/ebpf"
+	"github.com/cilium/cilium/pkg/types"
+)
+
+const (
+	MapMaxEntries = 1 << 14
+	MapName4      = "cilium_world_cidrs4"
+)
+
+// WorldCIDRKey4 is the key of a world CIDRs map.
+type WorldCIDRKey4 struct {
+	// PrefixLen is full 32 bits of mask bits
+	PrefixLen uint32
+	IP        types.IPv4
+}
+
+// WorldCIDRVal is the value of world CIDRs maps.
+type WorldCIDRVal struct {
+	Exists uint8
+}
+
+// wolrdCIDRsMap is the internal representation of a world CIDRs map.
+type worldCIDRsMap struct {
+	*ebpf.Map
+}
+
+var (
+	WorldCIDRsMap *worldCIDRsMap
+)
+
+// InitWorldCIDRsMap initializes the world CIDRs map.
+func InitWorldCIDRsMap() error {
+	return initWorldCIDRsMap(MapName4, true)
+}
+
+// OpenWorldCIDRsMap initializes the world CIDRs map.
+func OpenWorldCIDRsMap() error {
+	return initWorldCIDRsMap(MapName4, false)
+}
+
+// initworldCIDRsMap initializes the world CIDR map.
```

Comment:
Typo: `initworldCIDRsMap` -> `initWorldCIDRsMap`

Reply:
How did you even spot that? :exploding_head: